src/DurableTask.Core/TaskOrchestrationDispatcher.cs

```diff
@@ -34,6 +34,7 @@ namespace DurableTask.Core
     /// </summary>
     public class TaskOrchestrationDispatcher
     {
+        readonly DataConverter DataConverter;
```

Comment:
nit: field should be lower case
```suggestion
        readonly DataConverter dataConverter;
```